app/serializers/job_serializer.rb

```diff
@@ -1,6 +1,6 @@
 class JobSerializer < ActiveModel::Serializer
   attributes :id, :title, :description, :url, :location, :posted_date, :remote
-  has_one :company
+  has_one :company, serializer: CompanyForJobsSerializer
```

Comment:
@jeneve
I have separate serializers because if you only have one for each model and they each reference each other (via has_many/has_one), you get infinite circular references and "stack trace too deep."

With two, you can have the Jobs serializer reference that it belongs to a company, but use a different serializer for the company in that instance that doesn't list all its jobs (and vice versa).